lua/whitespace-nvim/init.lua

```diff
@@ -36,15 +30,28 @@ whitespace.trim = function()
   end
 end
 
+local function file_type()
+  if vim.tbl_contains(config.ignored_filetypes, vim.bo.filetype) then
+    whitespace.clean()
+    return
+  end
+  whitespace.highlight()
```

Comment:
Can you use `if ... else` here like in `term_open` please? That way it's more consistent